src/lib/dev-environment/dev-environment-core.js

```diff
@@ -303,3 +307,31 @@ export async function resolveImportPath( slug: string, fileName: string, searchR
 		dockerPath,
 	};
 }
+
+export async function importMediaPath( slug: string, filePath: string ) {
+	const resolvedPath = resolvePath( filePath );
+
+	if ( ! fs.existsSync( resolvedPath ) || ! fs.lstatSync( resolvedPath ).isDirectory() ) {
+		throw new Error( 'The provided path does not exist or it is not valid (see "--help" for examples)' );
+	}
+
+	const files = fs.readdirSync( resolvedPath );
+	if ( files.indexOf( uploadPathString ) > -1 ) {
+		const confirm = await prompt( {
+			type: 'confirm',
+			name: 'continue',
+			message: 'The provided path contains an uploads folder inside. Do you want to continue?',
```

Comment:
What is meant by this? Is it that we suspect user probably pointed to the wrong folder? Like `/path/to/uploads` to `/path/to` instead of `/path/to/uploads`?

Reply:
Exactly. We want to prevent data ending up in a path like `wp-content/uploads/uploads`. It's a warning, however, so user can decide to go ahead.

Reply:
Got it.. makes sense!